src/pages/saksbehandling/sakintro/Utbetalinger.tsx

```diff
@@ -1,93 +1,182 @@
+                                    </li>
+                                );
+                            })}
+                        </ol>
+                    </div>
+                    <div className={styles.utbetalingKnappContainer}>
+                        {kanStanses ? (
+                            <Fareknapp onClick={() => setModalOpen(true)}>
+                                {intl.formatMessage({ id: 'display.utbetalingsperiode.stoppUtbetaling' })}
+                            </Fareknapp>
+                        ) : (
+                            <Knapp
+                                onClick={() => {
+                                    if (
+                                        props.kanStansesEllerGjenopptas === KanStansesEllerGjenopptas.GJENOPPTA &&
+                                        !RemoteData.isPending(gjenopptaUtbetalingerStatus)
+                                    ) {
+                                        dispatch(
+                                            sakSlice.gjenopptaUtbetalinger({
+                                                sakId: props.sakId,
+                                            })
+                                        );
+                                    }
+                                }}
+                                spinner={RemoteData.isPending(gjenopptaUtbetalingerStatus)}
+                            >
+                                {intl.formatMessage({ id: 'display.utbetalingsperiode.gjenopptaUtbetaling' })}
+                            </Knapp>
+                        )}
+                    </div>
+                </div>
+                <ModalWrapper
+                    isOpen={modalOpen}
+                    closeButton={true}
+                    onRequestClose={() => setModalOpen(false)}
+                    contentLabel={'stansUtbetalinger'}
                 >
-                    Gjenoppta utbetalinger
-                </Knapp>
-            )}
-            {RemoteData.isFailure(stansUtbetalingerStatus) && (
-                <AlertStripe type="feil">Klarte ikke stanse utbetalingene.</AlertStripe>
-            )}
-            {RemoteData.isSuccess(stansUtbetalingerStatus) && (
-                <AlertStripe type="suksess">Utbetalingene er stanset.</AlertStripe>
-            )}
-            {RemoteData.isFailure(gjenopptaUtbetalingerStatus) && (
-                <AlertStripe type="feil">Klarte ikke gjenoppta utbetalingene.</AlertStripe>
-            )}
-            {RemoteData.isSuccess(gjenopptaUtbetalingerStatus) && (
-                <AlertStripe type="suksess">Utbetalingene er gjenopptatt.</AlertStripe>
-            )}
+                    <div className={styles.modalContainer}>
+                        <Undertittel>
+                            {intl.formatMessage({ id: 'display.utbetalingsperiode.stansUtbetalingerTil' })}
+                            {søker.navn.fornavn} {søker.navn.mellomnavn} {søker.navn.etternavn}
```

Comment:
Blir denna riktig? hur kommer det in space mellan navnen?

Reply:
🧙